conf/config-default.yaml

```diff
@@ -74,9 +74,10 @@ apisix:
   allow_admin:                  # http://nginx.org/en/docs/http/ngx_http_access_module.html#allow
     - 127.0.0.0/24              # If we don't set any IP list, then any IP access is allowed by default.
     #- "::/64"
-  #admin_listen:                # use a separate port
-  #  ip: 127.0.0.1              # Specific IP, if not set, the default value is `0.0.0.0`.
-  #  port: 9180
+  admin_listen:                 # use a separate port
+    ip: 0.0.0.0                 # Specific IP, if not set, the default value is `0.0.0.0`.
```

Comment:
should be better to use `127.0.0.1` as the default listen IP

Reply:
This PR doesn't change the default IP.
We already use `0.0.0.0` as the default IP of DP and CP now.